coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/GlobalAggregationCalculatorTest.kt

```diff
@@ -727,6 +727,18 @@ class GlobalAggregationCalculatorTest {
     assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(7u, 6u)).isEqualTo(6u)
     assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(11u, 6u)).isEqualTo(6u)
     assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(12u, 6u)).isEqualTo(12u)
+
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(1u, 9u)).isEqualTo(1u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(2u, 9u)).isEqualTo(2u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(3u, 9u)).isEqualTo(3u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(4u, 9u)).isEqualTo(4u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(5u, 9u)).isEqualTo(5u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(6u, 9u)).isEqualTo(6u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(7u, 9u)).isEqualTo(7u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(8u, 9u)).isEqualTo(8u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(9u, 9u)).isEqualTo(9u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(11u, 9u)).isEqualTo(9u)
+    assertThat(GlobalAggregationCalculator.getUpdatedAggregationSize(12u, 9u)).isEqualTo(9u)
```

Comment:
Maybe we can make it prettier with a function?

Reply:
Resolved with private helper function `assertOneToAggregationSizeUsesAggregationSize`